stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.1.0-a
+version: 1.2.0-a
```

Comment:
I probably should have sought guidance one this ahead of time. You're correct that I was trying to keep the version of the chart closely aligned with the version of Traefik it installs. What I did was append the `-a` on the end to reflect that this is was the first iteration of the Traefik v1.1.0 chart. Based on that, the correct version bump here would be to `1.1.0-b`.